saw-remote-api/python/saw/proofscript.py

```diff
@@ -9,6 +9,26 @@ class ABC(Prover):
   def to_json(self) -> Any:
     return { "name": "abc" }
 
+class ABC_Verilog(Prover):
+  def to_json(self) -> Any:
+    return { "name": "w4-abc-verilog" }
+
+class ABC_SMTLib(Prover):
+  def to_json(self) -> Any:
+    return { "name": "w4-abc-smtlib" }
+
+class ABC_SBV(Prover):
+  def to_json(self) -> Any:
+    return { "name": "sbv-abc" }
+
+class Boolector(Prover):
+  def to_json(self) -> Any:
+    return { "name": "w4-boolector" }
+
+class Boolector_SBV(Prover):
+  def to_json(self) -> Any:
+    return { "name": "sbv-boolector" }
```

Comment:
It's a bit strange that the `Boolector` class uses `"sbv-boolector"` instead of just `"boolector"`. I get that they currently map to the same thing, but isn't that a choice better left to the remote API rather than the Python bindings?

Reply:
Oh, yes. I agree. I'll fix that.